core/src/software/aws/toolkits/core/utils/StringUtils.kt

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.utils
+
+/**
+ * Designed to convert underscore separated words (e.g. UPDATE_COMPLETE) into title cased human readable text
+ * (e.g. Update Complete)
+ */
+fun String.toHumanReadable() = split('_').map { it.toLowerCase().capitalize() }.joinToString(" ")
```

Comment:
`=StringUtil.toTitleCase(this.lowercase().replace('_', ' '))` doesn't require creating arrays

Reply:
Roger-that